kickoff/jsonexport.py

```diff
@@ -0,0 +1,244 @@
+from kickoff import app
+
+from flask import jsonify, render_template
+
+from kickoff.model import getReleases
+from kickoff.firefoxversions import firefoxversions
+
+from kickoff.thunderbirddetails import primary_builds as tb_primary_builds, beta_builds as tb_beta_builds
+from kickoff.thunderbirdversions import thunderbirdversions
+
+from kickoff.mobileversions import mobileversions
+
+import config
+
+
+def getFilteredReleases(product, categories, ESR_NEXT=False, lastRelease=None, withL10N=False):
```

Comment:
This is still using custom regexes rather than ANY_VERSION_REGEX. I'm not r-'ing over this necessarily, but what's the rationale?